evm/src/contracts/CircleIntegration/Storage.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.22;
+
+/**
+ * struct State {
+ *         // state.slot := 0x0
+ *         uint16 chainId;
+ *         uint8 wormholeFinality;
+ *         uint32 localDomain;
+ *         address wormhole;
+ *         uint16 governanceChainId;
+ *
+ *         // state.slot := 0x1
+ *         bytes32 governanceContract;
+ *
+ *         // state.slot := 0x2
+ *         address circleBridgeAddress;
+ *
+ *         // state.slot := 0x3
+ *         address circleTransmitterAddress;
+ *
+ *         // state.slot := 0x4
+ *         address circleTokenMinterAddress;
+ *
+ *         // state.slot := 0x5
+ *         mapping(address => bool) initializedImplementations;
+ *
+ *         // state.slot := 0x6
+ *         mapping(uint16 => bytes32) registeredEmitters;
+ *
+ *         // state.slot := 0x7
+ *         mapping(uint16 => uint32) chainIdToDomain;
+ *
+ *         // state.slot := 0x8
+ *         mapping(uint32 => uint16) domainToChainId;
+ *
+ *         // state.slot := 0x9
+ *         mapping(bytes32 => bool) consumedMessages;
+ *
+ *         // state.slot := 0xa
+ *         uint256 evmChain;
```

Comment:
Same with this, it's not consuming a slot. You could move the below items to start from slot 0